app/controllers/admin/item_holds_controller.rb

```diff
@@ -2,7 +2,13 @@ module Admin
   class ItemHoldsController < BaseController
     def index
       @item = Item.find(params[:item_id])
-      @holds = @item.active_holds.order(created_at: :asc)
+      @holds = @item.holds.order(created_at: :asc)
+      @holds =
+        if params[:inactive]
+          @holds.inactive
+        else
+          @holds.active
+        end
```

Comment:
This is totally fine, although I find the flow easier to follow when we can avoid re-assigning the value of `@hold` like this:

```suggestion
      holds_scope = @item.holds.order(created_at: :asc)
      @holds =
        if params[:inactive]
          holds_scope.inactive
        else
          holds_scope.active
        end
```